norddrop/src/device/types.rs

```diff
@@ -123,6 +141,9 @@ pub struct Config {
     pub moose_event_path: String,
     pub moose_prod: bool,
     pub storage_path: String,
+
+    #[serde(rename = "checksum_events_size_threshold_bytes")]
```

Comment:
Can we actually name the field like this?

Reply:
In the codebase we for sure know we're ever talking in bytes, however I noticed in the API usage it sometimes raises the questions, thus I opted out for the best from both worlds.

It's however not a hard opinion, we can for sure have the same name in the codebase as well if you prefer

Reply:
Nah, it's fine. I was just curious why it's like that